src/inferences/junctionTree/create-initial-potentials.ts

```diff
@@ -86,10 +109,11 @@ const getPotentialValue = (combination: ICombinations, network: INetwork, given:
   return 0
 }
 
-const createCliquePotential = curry((clique: IClique, network: INetwork, given: ICombinations, cliqueFactors: ICliqueFactors, combination: ICombinations): ICliquePotentialItem => ({
-  when: combination,
-  then: getPotentialValue(combination, network, given, cliqueFactors[clique.id]),
-}))
+const createCliquePotential: (clique: IClique, network: INetwork, given: ICombinations, cliqueFactors: ICliqueFactors) => (combination: ICombinations) => ICliquePotentialItem =
```

Comment:
added an explicit type signature.   Some of the tests were failing to compile because they could not infer the correct type.